src/Automation/Automation.Test/SessionRecords/Commands.Automation.Test.UpdateManagementTests/CreateLinuxIncludeKbNumbersSUC.json

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
Did you mean to name the file 'CreateLinuxIncludedPackageNameMasksSUC.json' instead of 'CreateLinuxInvludeKbNumbersSUC.json' as here?

Reply:
Thank you. Fixed